src/blockdata/transaction.rs

```diff
@@ -703,18 +690,6 @@ mod tests {
         assert_eq!(txin.witness.len(), 0 as usize);
     }
 
-    #[test]
-    fn test_is_coinbase () {
```

Comment:
I think this test case also should stay here. It should be changed to something which doesn't use `genesis_block()`

Reply:
https://github.com/chaintope/rust-tapyrus/issues/16